sdk/src/client/api/block_builder/input_selection/core/transition.rs

```diff
@@ -44,11 +44,24 @@ impl InputSelection {
             return Ok(None);
         }
 
+        let mut highest_foundry_serial_number = 0;
+        for output in self.outputs.iter() {
+            if let Output::Foundry(foundry) = output {
+                if let ChainId::Foundry(foundry_id) = foundry.chain_id() {
```

Comment:
You don't need to go through the chain_id? just call `id()`

Reply:
Thanks, I searched for the wrong name and didn't found it

Reply:
Maybe we should rename it... I initially didn't do it because I wanted to show that it's computing it and not returning it. But yeah, not sure if we really care